FWCore/ParameterSet/interface/DocFormatHelper.h

```diff
@@ -13,43 +13,34 @@ namespace edm {
   class ParameterDescriptionNode;
 
   class DocFormatHelper {
-
   public:
-
-    enum DescriptionParent {
-      TOP,
-      OR,
-      XOR,
-      AND,
-      OTHER      
-    };
-
-    DocFormatHelper() :
-      brief_(false),
-      lineWidth_(80),
-      indentation_(4),
-      startColumn2_(24U),
-      section_(),
-      pass_(0),
-      column1_(0),
-      column2_(0),
-      column3_(0),
-      counter_(0),
-      parent_(OTHER)
-    { }
+    enum DescriptionParent { TOP, OR, XOR, AND, OTHER };
+
+    DocFormatHelper()
+        : brief_(false),
+          lineWidth_(80),
+          indentation_(4),
+          startColumn2_(24U),
+          section_(),
+          pass_(0),
+          column1_(0),
+          column2_(0),
+          column3_(0),
+          counter_(0),
+          parent_(OTHER) {}
 
     void init();
 
-    bool brief() const { return brief_; }
-    size_t lineWidth() const { return lineWidth_; }
-    int indentation() const { return indentation_; }
-    int startColumn2() const { return startColumn2_; }
+    bool brief() const { return brief(); }
+    size_t lineWidth() const { return lineWidth(); }
+    int indentation() const { return indentation(); }
+    int startColumn2() const { return startColumn2(); }
```

Comment:
these changes do not look right, these function will just call themselves forever ?